web/layouts/rules/rule.html

```
@@ -24,56 +24,56 @@
                 <ol class="flex items-center space-x-4">
                     <li>
                         <div class="flex">
-                            <a href="{{ relURL "/" }}" class="text-sm font-medium text-gray-400 hover:text-gray-200">
-                                <svg class="w-4 h-4" xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 24 24" stroke="currentColor" stroke-width="2">
+                            <a href="{{ relURL "/" }}" class="text-sm font-medium text-pc-grey-950 hover:text-pc-green-hover">
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ < ].

(spec-char-escape)